src/components/MessageInput/hooks/messageInput.js

```diff
@@ -208,7 +208,7 @@ function messageInputReducer(state, action) {
 }
 /**
  * hook for MessageInput state
- * @param {Props} props
+ * @type{import('types').useMessageInput}
```

Comment:
```suggestion
 * @type{import('types').useMessageInputState}
```

Reply:
I started like that but noticed we export it named `useMessageInput` here: https://github.com/GetStream/stream-chat-react/blob/316cf2958e4fa81114beccd8a4753758404c4081/src/components/MessageInput/hooks/index.js

Reply:
oh right, should we rename it to `useMessageInput` so it's more consistent?

Reply:
👍  Did it: https://github.com/GetStream/stream-chat-react/pull/601/commits/c3f981ac2563d2a30dde9903fded33cc524e311e

Reply:
Nice